<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>second</class>
 <widget class="QWidget" name="second">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>593</height>
   </rect>
  </property>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>470</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Температура С</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Всего анионов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Минералзация, TDS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Стронций</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Сульфат</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>мэк/л</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Бикарбонат</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_28">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_31">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_36">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_40">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Нитрат</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_25">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>480</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_38">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Всего катионов</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_39">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Хлорид</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>мэк/л</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_21">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_32">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>540</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_34">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>420</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Силикат йон</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Установка pH</string>
   </property>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>pH</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_41">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>40</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Коррекция</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Без коррекции</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>CO3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>CO2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_42">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_43">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Натрий</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_35">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>560</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_14">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>380</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Марганец</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Алюминий</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_23">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>440</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_13">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Йон</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_24">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_15">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>470</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_22">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Магний</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>мг/л</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_18">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_27">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_29">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_19">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_11">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Калий</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Йонная сила</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_20">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>400</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Барий</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Фосфат</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>мг/л</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Железо</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Йон</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_33">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_37">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>460</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Кальций</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_30">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Фторид</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_16">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_26">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Выбор антискаланта</string>
   </property>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ppm</string>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>411</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Обработка антискалантом</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
